projects/1.19/assets/dungeons-gear/dungeons_gear/lang/zh_cn.json

```
@@ -0,0 +1,1066 @@
+
+  "item.dungeons_gear.coral_blade": "珊瑚之刃",
+  "lore.dungeons_gear.coral_blade": "珊瑚之刃能够精准地刺穿敌人。",
+  "item.dungeons_gear.sponge_striker": "海绵刺刃",
+  "lore.dungeons_gear.sponge_striker": "这把刀看起来色彩黯淡且死气沉沉，但它可以在战斗中吸收能量，并以强大的爆发力将其释放。",
+
+  "item.dungeons_gear.anchor": "船锚",
+  "lore.dungeons_gear.anchor": "那些强大到足以在战斗中挥舞船锚的人遵循了传说中航海勇士的传统。",
+  "item.dungeons_gear.encrusted_anchor": "覆藤船锚",
+  "lore.dungeons_gear.encrusted_anchor": "这个覆藤船锚已经在海上遗失了许久，经过漫长岁月，它已磨得犀利和带有腐蚀性。",
+
+  "ability.dungeons_gear.boosts_attack_speed": "增加攻击速度",
+  "ability.dungeons_gear.busy_bee": "概率召唤蜜蜂（忙碌蜜蜂 I）",
+  "ability.dungeons_gear.chains": "禁锢目标（禁锢 I）",
+  "ability.dungeons_gear.committed": "增加对敌对生物造成的伤害（进伤 I）",
+  "ability.dungeons_gear.continuous_attacks": "快速攻击",
+  "ability.dungeons_gear.critical_hit": "增加暴击的命中率（暴击 I）",
+  "ability.dungeons_gear.deals_absorbed_damage_during_attack_combo": "攻击时获得伤害抗性",
+  "ability.dungeons_gear.echo": "概率连续攻击两次（回击 I）",
+  "ability.dungeons_gear.exploding": "击杀生物后使其爆炸（爆炸 I）",
+  "ability.dungeons_gear.fast_thrusts": "快攻",
+  "ability.dungeons_gear.fire_aspect": "灼烧生物（火焰附加 I）",
+  "ability.dungeons_gear.fortune": "生物掉落更多战利品（抢夺 I）",
+  "ability.dungeons_gear.freezing": "降低生物移速（霜冻 I）",
+  "ability.dungeons_gear.great_splash": "溅射伤害",
+  "ability.dungeons_gear.illagers_bane": "增加对灾厄村民造成的伤害（灾厄杀手 I）",
+  "ability.dungeons_gear.knockback": "强力击退（击退 I）",
+  "ability.dungeons_gear.leeching": "击杀生物恢复生命值（吸血 I）",
+  "ability.dungeons_gear.prospector": "生物掉落更多的稀有战利品（探矿者 I）",
+  "ability.dungeons_gear.radiance": "治疗区域内的友方目标（治愈光辉 I）",
+  "ability.dungeons_gear.rampaging": "增加攻击速度（狂怒 I）",
+  "ability.dungeons_gear.rapid_slashes": "快速攻击",
+  "ability.dungeons_gear.relentless_combo": "无情连击",
+  "ability.dungeons_gear.reliable_combo": "节奏连击",
+  "ability.dungeons_gear.rushdown": "击杀生物后获得短暂的速度加成（冲刺 I）",
+  "ability.dungeons_gear.sharpness": "额外伤害（锋利 I）",
+  "ability.dungeons_gear.shockwave": "暴击施放冲击波（冲击波 I）",
+  "ability.dungeons_gear.smite": "增加对亡灵生物造成的伤害（亡灵杀手 I）",
+  "ability.dungeons_gear.soul_siphon": "概率获得灵魂（灵魂虹吸 I）",
+  "ability.dungeons_gear.spin_attack": "旋转攻击",
+  "ability.dungeons_gear.stunning": "眩晕生物（眩晕 I）",
+  "ability.dungeons_gear.stylish_combo": "增加在第三、六、八次和最后一击的伤害",
+  "ability.dungeons_gear.swirling": "概率产生范围伤害（旋涡 I）",
+  "ability.dungeons_gear.thrust_attack": "突袭",
+  "ability.dungeons_gear.thundering": "生成雷电（引雷 I）",
+  "ability.dungeons_gear.weakening": "削弱目标生物的攻击伤害（虚弱 I）",
+  "ability.dungeons_gear.wind_up_attack": "发条射击",
+
+  "entity.dungeons_gear.soul_wizard": "灵魂巫师",
+
+  "item.dungeons_gear.arrow_bundle": "箭捆",
+  "ability.dungeons_gear.arrow_hoarder": "+1 每束箭矢（箭矢存储 I）",
+
+  "dungeons_gear.gilded": "镀金",
+
+  "death.ice_chunk": "%1$s 被冰块击碎",
+  "death.attack.ice_chunk": "%1$s 被 %2$s 的冰块击碎",
+
+  "subtitles.entity.soul_wizard.appear": "灵魂巫师：生成",
+  "subtitles.entity.soul_wizard.hurt": "灵魂巫师：受伤",
+  "subtitles.entity.soul_wizard.shoot": "灵魂巫师：射击",
+  "subtitles.entity.soul_wizard.projectile_impact": "灵魂巫师：球体撞击"

```

```suggestion
  "subtitles.entity.soul_wizard.projectile_impact": "灵魂巫师能量球：撞击"
```
是 `Soul Wizard Orb` `impacts` 而不是 `Soul Wizard` `Orb impacts`

---

谢谢你的建议，已更改。